### PR TITLE
StoreBehaviour and StoreConnector

### PR DESCRIPTION
**Pull request by Hartorn**
### Description
@pierr This PR is correcting the connect.js, so it avoids setState when the component is not mounted
Also, it adds a new component to connect component to a Store, just with props.

Maybe some more testing is needed, but it seems to work fine.

### CODE REVIEW

File: src/components/store-connector/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    component: PropTypes.func.isRequired
+};
+// defaultProps
+const defaultProps = {
+};
+//displayName
+const displayName = 'StoreConnector';
+
+const StoreConnector = ({storeProperties, storeGetFunction, component, ...others}) => {
+    const ConnectedComp = connectToStore(storeProperties, storeGetFunction)(component);
```
Comment (pierr) [inline src/components/store-connector/index.js]: so much nicer than the previous version

File: src/components/store-connector/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import React, {PropTypes} from 'react';
+import connectToStore from 'focus-components/behaviours/store/connect';
```
Comment (pierr) [inline src/components/store-connector/index.js]: it should be a local require....

File: src/components/store-connector/index.js (hunk 1 above)
Comment (pierr) [inline src/components/store-connector/index.js]: @Hartorn What you are doing is an other pattern, you can have :

- high order component which is `connectToStore`
- child function component which is the `component` props

These pattern are both correct but i am not comfortable with enforcing two way of doing the same thing.
Even if they are even regarding their behaviour.

File: src/behaviours/store/connect.js [hunk 3]
```diff
@@ -69,9 +69,20 @@ export default function connectToStores(storesConfiguration, getState) {
                 });
             }
 
+			componentDidMount(){
+				this._isMounted = true;
```
Comment (pierr) [inline src/behaviours/store/connect.js]: Nice to respect the good react pattern https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html
Thanks!

**Comment by Hartorn** [comment]
This is out of date

**Pull request closed without merging**